### ISSUE TITLE
Add white icons for dark mode desktops

### ISSUE THREAD

**Issue by mig8447** [iXab3r/MicSwitch#41]
The app should be able to detect the OS theme and offer white icon versions

**Comment by rday74** [comment]
[Mic_Icons.zip](https://github.com/iXab3r/MicSwitch/files/6727269/Mic_Icons.zip)

It's not automatic but you can use these ones.

**Comment by iXab3r** [comment]
Hi, I am currently working on another project that will have media library as a feature. As soon as I will finish it there, I will be able to allow user to select one of predefined icons as well as upload custom ones (uploading already works, but with media library you'll be able to select icon you've uploaded earlier at any moment.

Automatically detect theme is not really a reliable way in windows because there are literally thousands of themes and most of them do not report whether they are "light" or "dark" correctly.

**Comment by mig8447** [comment]
Fair enough, I think as long as the app ships a pair of black and a pair of black icons everything should be smooth 😄

**Comment by danialasghar** [comment]
Is changing the icon supposed to be possible in Windows? I've tried clicking on it multiple times but had no luck

**Comment by DGrothe-PhD** [comment]
If you like to sometimes switch back to light mode, you may try these blueish icons
[MicIconsNavy.zip](https://github.com/iXab3r/MicSwitch/files/11788309/MicIconsNavy.zip)
